server/utils/helpers.js

```diff
@@ -81,7 +84,7 @@ export const hashPassword = password => bcrypt.hashSync(password, 10);
  * @returns {boolean} match?
  */
 export const comparePassword = (password, hash) =>
-  bcrypt.compareSync(password, hash);
+  bcrypt.compareSync(hash, password);
```

Comment:
In the `bcrypt` documentation, `password` is the first argument. You seem to have swapped them here.

Reply:
I have done that